api.model.js

```diff
@@ -0,0 +1,11 @@
+const db = require('./db/connection')
+
+exports.topicFinder = () => {
+    const query = 'SELECT * FROM topics;';
+
+    return db.query(query).then((result)=>{
+        console.log(result.rows)
```

Comment:
remember to remove un-necessary console logs or comments before making a pull request.